### DISCUSSION TITLE
能否给出azure openai配置的详细样例？特别是API URL Path应该填写什么？谢谢

### DISCUSSION THREAD

**Discussion by lgxlg2003** [nextai-translator/nextai-translator#599] [Q&A]
能否给出azure openai配置的详细样例？特别是API URL Path应该填写什么？谢谢

**Comment by chengjunjian** [comment]
这个确实要说明下啊

**Comment by freecolour** [comment]
同等说明，谢谢

**Comment by XunWind** [comment]
我摸索下已经配置成功 

打开Azure找到密钥和终结点 一图抵千言：

![image](https://user-images.githubusercontent.com/23732553/234778129-e4cc347a-29ff-486b-a357-df89d9a1fbe3.png)
![image](https://user-images.githubusercontent.com/23732553/234778445-45f8ade8-0d93-46ec-961c-1068bf668a72.png)

固定的API URL路径: /openai/deployments/{部署模型你起的名称}/chat/completions?api-version=2023-03-15-preview

**Reply by itcodes** [reply]
> 我摸索下已经配置成功
> 
> 打开Azure找到密钥和终结点 一图抵千言：
> 
> ![image](https://user-images.githubusercontent.com/23732553/234778129-e4cc347a-29ff-486b-a357-df89d9a1fbe3.png) ![image](https://user-images.githubusercontent.com/23732553/234778445-45f8ade8-0d93-46ec-961c-1068bf668a72.png)
> 
> 固定的API URL路径: /openai/deployments/{部署模型你起的名称}/chat/completions?api-version=2023-03-15-preview

你这个API URL怎么摸索出来的？

**Reply by rooly2022** [reply]
为什么我改成 06-13 的模型就无法使用了？你怎么处理的？感谢！

**Reply by brucmao** [reply]
我也是，0613模型应该怎么填啊

**Reply by brucmao** [reply]
原来少了一个chat,😂
openai/deployments/{部署模型你起的名称}/**chat**/completions?api-version=2023-03-15-preview

**Reply by smile-xuc** [reply]
原来真的就是少了一个/chat/，实测有效。

**Reply by keyknight23** [reply]
可以在下面这个链接查看可用的api-version
https://learn.microsoft.com/en-US/azure/ai-services/openai/reference

**Reply by z6m8n7ljw** [reply]
好像这个方式又失效了
![Screen Shot 2023-11-28 at 15 27 13](https://github.com/openai-translator/openai-translator/assets/17857021/20b7b126-fa16-4161-a39a-b39da62169b9)
![Screen Shot 2023-11-28 at 15 28 28](https://github.com/openai-translator/openai-translator/assets/17857021/2d2744ab-30f7-4c1a-b374-905d591d7da9)

**Reply by joeaki1983** [reply]
我的也不行，各种软件对azure的配置都很不友好，头很大。

**Reply by joeaki1983** [reply]
已经搞定，要在azure建立ChatGPT3.5 0301才行，更新的版本都不行。

**Reply by z6m8n7ljw** [reply]
今天再试，如我上述回复的这个方式又 work 了
![image](https://github.com/openai-translator/openai-translator/assets/17857021/a186b34d-5d41-4360-b4c9-61d07f7a51ff)

**Comment by freecolour** [comment]
多谢，照着做果然搞定了

**Reply by leegx1998** [reply]
加上 chat 就行了？

**Comment by 716lufei** [comment]
十分感谢，搞定了

**Comment by 716lufei** [comment]
对 Azure OpenAI API 版本 2023-03-15-preview 下的 Chatcompletions_Create 操作的请求已超出当前 OpenAI S0 定价层的调用速率限制。 请5秒后重试。 如果您想进一步提高默认费率限制，请访问此处：https://aka.ms/oai/quotaincrease。
能通了又头疼关了哈哈

**Comment by 716lufei** [comment]
搞定了，从新创建了一下，然后把配额增加了一下

**Comment by jasonzhouu** [comment]
现在azure openai改版了，url可以在azure ai studio的部署页面找到，把URL域名和path分开输入到openai translator的设置里就行了




<img width="1440" alt="图片" src="https://github.com/user-attachments/assets/562a8bc9-02be-41ac-bfce-3851fcab1ce5">

<img width="620" alt="图片" src="https://github.com/user-attachments/assets/01f96b12-a311-4410-8adf-a7f47ac6c386">
